libs/components/lookup/src/lib/modules/search/search.component.ts

```diff
@@ -320,10 +346,25 @@ export class SkySearchComponent implements OnDestroy, OnInit, OnChanges {
     this.#changeRef.markForCheck();
   }
 
-  #searchShouldCollapse() {
+  #searchShouldCollapse(): boolean {
     return (
       (this.isCollapsible || this.isCollapsible === undefined) &&
       this.isFullWidth !== true
     );
   }
+
+  #setupSearchChangedEvent(): void {
+    this.#searchUpdatedUnsubscribe.next();
+
+    this.#searchUpdated
+      .asObservable()
```

Comment:
This `asObservable()` isn't needed here. You'd only use that if you were exposing the `Subject` to an API consumer and you want to obscure the underlying `Subject` methods like `next()`.